move_zeroes_to_end.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <stdlib.h>
```

Comment:
Move this code to the correct folder. Use `cstdlib`. In all for loops, use preincrement.